content/en/docs/tasks/configure-pod-container/user-namespaces.md

```diff
@@ -0,0 +1,214 @@
+
+{{< codenew file="pods/user-namespaces-stateless.yaml" >}}
+
+1. Create the pod on your cluster:
+
+   ```shell
+   kubectl apply -f https://k8s.io/examples/pods/user-namespaces-stateless.yaml
+   ```
+
+1. Attach to the container and run `cat /proc/self/uid_map`:
+
+   ```shell
+   kubectl attach -it userns bash
+   ```
+
+And run the command. The output is similar to this:
+
+```none
+root@userns:/# cat /proc/self/uid_map
+         0     131072      65536
+```
+
+This means that UID 0 is mapped to UID 131072 in the host and this is done for
+65536 consecutive UIDs. In other words, UID 1 in the container is mapped to UID
+131072+1 on the host, and so on.
+
+For more details, please see `man 7 user_namespaces`.
+
+## Verify if user namespaces is active for a pod
+
+To verify that a pod is using user namespaces, if you exec into the pod you can
+run:
+
+```none
+root@userns:/# cat /proc/self/uid_map
+         0     131072      65536
+root@userns:/# cat /proc/self/gid_map
+         0     131072      65536
+```
+
+User namespaces are active if the last value is 65536 in both cases. If you see
+that the last value is 4294967295, this means user namespaces are not active.
+
+The value 4294967295 is the max number of UIDs/GIDs a Linux system can use. If
+you see the whole range of UIDs/GIDs available to the pod, it means a user
+namespaces is not being used.
+
+If your pod's containers doesn't have a shell, you can do it from the host.
+
+If you know which PID in the host the container process is using, you can read
+from the host the same files: `cat /proc/$pid/uid_map` and `cat
+/proc/$pid/gid_map`, replacing `$pid` with the container PID.
+
+There are others ways that can be used to check if user namespaces are active or
+not. This section is not intended to list them all.
+
+## Limitations
+
+When using a user namespace for the pod, it is disallowed to use other host
+namespaces. In particular, if `hostUsers: false` then it is not allowed to set
+either of:
+
+ * `hostNetwork: true`
+ * `hostIPC: true`
+ * `hostPID: true`
+
+The pod is allowed to use no volumes at all or, if using volumes, only these
+volume types are allowed:
+
+ * configmap
+ * secret
+ * projected
+ * downwardAPI
+ * emptyDir
+
+To guarantee that the pod can read the files of such volumes, volumes are
+created as if `pod.spec.securityContext.FSGroup` is specified as `0`. If it is
+specified to a different value, this other value will of course be honored
+instead.
+
+As a by-product of this, folders and files for these volumes will have
+permissions for the group, even if defaultMode or mode to specific items of the
+volumes were specified without permissions to groups. For example, it is not
+possible to mount these volumes in a way that its files have permissions only
+for the owner. This is a limitation imposed by FSGroup. 
+
+This is a limitation in Kubernetes v1.25 and future versions of Kubernetes might
+remove this limitation when user namespaces are in use.
```

Comment:
Please [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future).

Reply:
Removed